timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,141 @@
+with your Timescale Forge VPC.
+
+Timescale Forge uses a different DNS name for a Timescale service once it has been attached
+to a VPC. This means that you will need to update your connection string if migrating a service
+from the public internet into a VPC, or vice-versa.
+</highlight>
+
+## Setup
+Before you begin, log in to the Timescale Forge console.
+
+### Create a new VPC
+In the Timescale Forge console, click `VPC` in the left navigation bar to go to the VPC
+dashboard. You can add new VPCs here for your Timescale Forge services to attach to.
+The VPCs created here are peered with your own VPC as part of the setup process.
+
+<img class="main-content__illustration" src="vpc-dashboard.png" alt="Navigate to the VPC dashboard in the console"/>
+
+Click `Create VPC`, type a name for your new VPC, and provide an IPv4 CIDR block.
+Make sure that the CIDR block you choose for your VPC does not overlap with the
+cloud VPC you are using to create a peering connection. If the CIDR blocks overlap,
+the peering process will fail. You can always find the CIDR block of your AWS VPC
+from the AWS console.
+
+<img class="main-content__illustration" src="create-vpc.png" alt="Create a new Forge VPC"/>
+
+<highlight type="tip">
+Though VPC peering is still covered by the Timescale Forge trial period, you will be required
+to enter a valid payment method in order to create a VPC.
+</highlight>
+
+### Create a peering connection
+When you have created a Timescale Forge VPC, you are ready to create a peering connection
+between your Forge VPC and your cloud VPC. To do this, expand the dropdown menu by clicking
+the `VPC Peering` column on your Forge VPC.
+
+<img class="main-content__illustration" src="create-peering-connection.png" alt="Expand the VPC Peering dropdown menu and enter info"/>
+
+From this menu, enter the AWS Account ID, AWS VPC ID and AWS region of your
+cloud VPC for the new peering connection. When you have entered the correct
+information, click `Add peering connection` to begin the peering process.
+
+This process is asynchronous, and results in a peering connection sent to your
+AWS account for you to accept. This is an important safety mechanism, never
+accept a peering connection from an unknown account. All Timescale Forge
+peering connections are sent from the AWS account `142548018081`.
+
+<highlight type="tip">
+Make note of the peering connection ID (starting with `pcx-`) as it is used in the next step.
+</highlight>
+
+<img class="main-content__illustration" src="aws-accept-peering-connection.png" alt="Accept peering connection in AWS console after verifying requestor account number"/>
+
+### Network Route & Security
+Once you have accepted the peering connection from the last step, the two VPCs will now
+be peered; however, in order to use this peering connection, you will need to update your
+VPC's route table to include the CIDR block of your peered Forge VPC, and lastly you will
```

Comment:
```suggestion
VPC's route table to include the CIDR block of your peered Forge VPC. You also
```

Reply:
When you use "will" in a sentence, you are flipping into future tense, which has a tendency to weird language. Where there's a will, you need to re-write 😉

Reply:
Excellent pointer, thank you!